tevm/docs/state/interfaces/StateManager.md

```
@@ -6,7 +6,11 @@
 
 # Interface: StateManager
 
+<<<<<<< HEAD
```

markdownlint-cli2 (0.18.1)

Heading style
Expected: atx; Actual: setext

(MD003, heading-style)